docs/submission/submitted-data.md

```diff
@@ -118,6 +119,10 @@ If you are logged in, the [File Repository](https://platform.icgc-argo.org/repos
  - **My Program Access**: These files are visible to [Program Members](/docs/submission/managing-program-access) only.  
  - **Full Member Access**: These files are visible to both the [Program Members](/docs/submission/managing-program-access) that submitted the data and to other ARGO Program Members that are part of Full Member programs. 
  - **Associate Member Access**: These files are visible to both the [Program Members](/docs/submission/managing-program-access) that submitted the data and to other ARGO Program Members that are part of Associate Member programs. 
- - **Released to the Public**: These files are publicly visible to any user that visits the ARGO Data Platform file Repository.  If the file is `controlled acccess` DACO permissions are required to download the file. If the file is `open` access, it can be downloaded anonymously. 
+ - **Released to the Public**: These files are publicly visible to any user that visits the ARGO Data Platform file Repository.  
+ 
+ ## DACO Access Control
```

Comment:
@joneubank I had made some changes to this section in my latest PR that will hopefully be released next sprint.... https://github.com/icgc-argo/argo-docs/pull/463

Will this create a merge conflict?

Reply:
Will probably conflict. Once that one is merged I will add my changes in since these are small.